tests/SConscript

```diff
@@ -2,4 +2,5 @@
 import os
 from lsst.sconsUtils import scripts, env
 
-scripts.BasicSConscript.tests()
+# testCoadds.py must be entirely skipped, and Scons doesn't like that
+scripts.BasicSConscript.tests(ignoreList=['testCoadds.py'])
```

Comment:
I am very confused. Why does scons care if tests are skipped? SconsUtils just runs the tests and looks at the exit status. I didn't think that exit status was bad if all tests are skipped (assuming you are skipping them properly).